Source/VCGeneration/VCGen.cs

```diff
@@ -729,7 +732,13 @@ private void ConvertCFG2DAGStandard(Implementation impl, Dictionary<Block, List<
                 b = new LoopInitAssertCmd(c.tok, c.Expr, c);
               }
 
-              b.Attributes = c.Attributes;
+              b.Attributes = (QKeyValue)c.Attributes?.Clone();
```

Comment:
Yes, it's necessary, since the `id` attribute on the new command will have a suffix appended. In principle, I guess the two commands could share an attribute list tail, but that seems more complex than it's worth.

Reply:
Note that the original command is discarded, so you could get away with one less clone, but either is OK with me.